lut3d_utils/lut3d_util.py

```diff
@@ -152,8 +171,8 @@ def read_from_cube_file(self, src):
       )
       return False
     self.lut_size = lut3d_size
-    self.lut_value = data
-
+    self.lut_value =  [data[self.shuffle_indices(i)]
```

Comment:
nit. There is an extra space in front of the first [

Reply:
Fixed. Thanks